nflow-engine/src/main/java/com/nitorcreations/nflow/engine/listener/AbstractWorkflowExecutorListener.java

```diff
@@ -0,0 +1,49 @@
+package com.nitorcreations.nflow.engine.listener;
+
+import com.nitorcreations.nflow.engine.workflow.definition.NextAction;
+
+/**
+ * AbstractWorkflowExecutorListener implement WorkflowExecutorListener with no-op/defaults
+ * method implementations. A subclasses can override just the methods it is interested in.
+ */
+abstract public class AbstractWorkflowExecutorListener implements WorkflowExecutorListener {
```

Comment:
Why is this class abstract - it has no abstract methods. Should it be a concrete class named WorkflowExecutorListenerAdapter ?